concepts/collateral-breakdown.md

```diff
@@ -0,0 +1,19 @@
+# Collateral Breakdown
+
+Collateral assets held by the Maker Protocol or its SubDAOs can be broken up into the following categories.
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Decentralized collateral
```

Comment:
I would maybe bold or reformat these slightly so that the subdivisions are more clear.